### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by celiopy** [OkuBrowser/oku#319] [Ideas]
Hello, I found this on flathub and I really like it!

I'm not 100% sure if this is the proper place to ask, but I'm willing to help with translations, if necessary. But as I didn't find anything related, I guess it still isn't possible?

I can translate it to pt (-BR).

Anyhow, I'm impressed, thank you for your work with this browser!

**Comment by emmyoh** [comment]
Thank you for volunteering! I need to plan out how I'll begin implementing translations and what the standard practices are. I'll get back to you when that's ready

**Reply by emmyoh** [reply]
This is now [on the roadmap](https://github.com/OkuBrowser/oku/discussions/330).
